### PR TITLE
Yobson

### PR DESCRIPTION
**Pull request by yobson**
Consider this both a spell/grammar check, and an idea check :)

### CODE REVIEW

**Comment by kraileth** [comment]
Just skimmed over it real quick - very exciting stuff! I also like details such as using toml as the configuration format. Will read again through it more carefully tomorrow morning.

**Comment by yobson** [comment]
Apologies for the force pushes. My email address was wrong on my git configuration on my mac so the GPG check was failing!

File: docs/plan/Proposal.tex [hunk 1]
```diff
@@ -120,6 +122,64 @@ \section{Proposal}
...
+\subsubsection{Logging and alarms}
+All output of the services will be redirected back to \texttt{utlosed} which will both write to a rotating log
+(configurable) and broadcast. The channel will be configurable, but the default will be: \texttt{logs/hostname/service}.
+This means that any logging client can filter for logs, logs from a specific host, and logs from a specific service on a specific
+host.
+
+Logs written to \texttt{stderr} (and crash reports detected by \texttt{utlosed}) will also be reported on \texttt{alarms/hostname/service}
+allowing for a service to listen specifically for things that are going wrong.
+
+All logs currently held on disk may be requested via the request responce socket interface. This allows remote clients to both
```
Comment (kraileth) [inline docs/plan/Proposal.tex]: Typo: _responce_

File: docs/plan/Proposal.tex [hunk 2]
```diff
@@ -120,6 +122,64 @@ \section{Proposal}
...
+respond live events happening (via subscription), but doesn't have to be running 100\% of the time to get a full history.
```
Comment (kraileth) [inline docs/plan/Proposal.tex]: 1. Should probably be "respond **to** live events"?
2. Also I'm not sure about the "doesn't". It refers to the remote clients, right? Them it should probably be "but those don't have to be running".

**Review by kraileth — APPROVED** [review]
Just three small things I noticed. When it comes to the actual content, I'd be happy to see this merged. There are a few things I want to further comment on; will do so via mail as time permits.

**Comment by yobson** [comment]
Should be okay now! Looking forward to the email

**Pull request merged**